src/CodeReview.Orchestrator/Services/ContainerService.cs

```diff
@@ -78,6 +78,9 @@ public async Task PullImageAsync(string imageName)
             await client.Images.CreateImageAsync(parameters, null, new NullProgress());
         }
 
+        public Task<string> CreateContainerAsync(string imageName, params MountedVolume[] volumes)
```

Comment:
To my mind we can inline this method. There only two usages of it in code. I think one method would make more sense.
It's up to you to decide if you want to leave it or not. If you like it you can leave it =)

Reply:
yeah, I like my approach. I will leave as it is, if you do not mind.